### ISSUE TITLE
Which part of the code uses the test set to evaluate the model results?

### ISSUE THREAD

**Issue by learnmat** [DaiShiResearch/TransNeXt#28]
**Thank you for writing such a novel paper.** 
I need to **ask a question**: according to the dataset format recommended by the author in the readme.md, I found that the data was not tested on the test set, only the training set and the validation set, **did the author use the test set? What part of the code is this set up?** Can you give me some pointers that I want to use on my dataset but can't find where to use the test set? Thank you
